### PR TITLE
Remove unnecessary data from node details #97

### PR DESCRIPTION
**Pull request by SzajekSzymon**
## Description
#111 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

---
I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

**Comment by tomaszmichalak** [comment]
Please check this requirement:
- singleNode - should not has obvious info about subtasks which alway are empty

File: src/js/helpers/graph/declarationHelper.js [hunk 1]
```diff
@@ -32,14 +32,27 @@ export const getNodeGroup = (node) => {
...
-const createVisNode = (node) => ({
-  id: node.id,
-  label: node.label,
-  group: getNodeGroup(node),
-  info: {
-    ...node.info,
-  },
-});
+const createVisNode = (node) => {
+  const info = node.status.toLowerCase() === 'missing'
```
Comment (tomaszmichalak) [inline src/js/helpers/graph/declarationHelper.js:36]: Missing node is always SINGLE.

I think we need to handle all SINGLE nodes (not only missing ones).

Comment (SzajekSzymon) [reply src/js/helpers/graph/declarationHelper.js:36]: for missing node we set custom log. For single we delete subtasks. So we need two diffrent functionality. we can only rewrite this to 

```
if single node {
    if missing node
       do something
    else
        do something
}
```

But i don't know this notation is better. Result is the same.

**Comment by tomaszmichalak** [comment]
We need to validate CI.

**Comment by Voycawojka** [comment]
Looks like CI is working now

File: src/js/helpers/graph/declarationHelper.js (hunk 1 above)
Comment (tomaszmichalak) [reply src/js/helpers/graph/declarationHelper.js:36]: ok

**Pull request merged**